scripts/run-integration-tests.sh

```diff
@@ -5,4 +5,8 @@ set -euo pipefail
 ./build/downloads/bin/conjure-verification-server ./build/resources/verification-server-test-cases.json ./build/resources/verification-server-api.conjure.json &
 SERVER_PID=$!
 yarn karma start --single-run --browsers ChromeHeadless karma.conf.js
-kill -kill ${SERVER_PID}
\ No newline at end of file
+
+if ps -p ${SERVER_PID} > /dev/null
+then
+    kill -kill ${SERVER_PID}
+fi
```

Comment:
Added this to make sure that the server process is still running before we kill it. Otherwise, the script would terminate with an error code if the server process was not not running.